my-app/src/components/thumbNails.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+function ThumbNails(props){
+    var {nextLink, classId, filePath, caption} = props;
+    return (
+        <div> 
+            <figure>
+                <a href ={nextLink}>
+                    <img className = {classId} src = {filePath} alt="" width="500" height="350" />
```

Comment:
lets not make width and height static make them prop variables too. You can make them default to 500 and 350 
{width || 500} 

(I think that works not fully sure)

We also don't scale the pictures properly but Luke's ticket doesn't do it either so I think its fine we can fix it if its a problem

Reply:
Just fix that then u can "Squash and Merge"